pkgs/development/python-modules/google-cloud-artifact-registry/default.nix

```diff
@@ -1,27 +1,38 @@
-{ buildPythonPackage
+{ lib
+, buildPythonPackage
 , fetchPypi
 , google-api-core
+, google-auth
 , grpc-google-iam-v1
-, lib
+, proto-plus
+, protobuf
 , pytestCheckHook
 , pythonOlder
+, setuptools
 }:
```

Comment:
nit: let's keep these alphabetical

Reply:
If we start with `lib` the list of inputs can be sorted alphabetically with most editors without additional effort. Also, starting with `lib` is kind of common practice for Python modules here. 

But the maintainer has the last word :wink: